client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/upgrade-modal.tsx

```diff
@@ -58,37 +58,21 @@ const UpgradeModal = ( { slug, isOpen, closeModal, checkout }: UpgradeModalProps
 			text: (
 				<p>
 					{ translate(
-						'You can purchase a subscription to use this theme or join the Premium plan to get it for free.'
+						"Gain access to all our Premium themes, and a ton of other features, with a subscription to the Premium plan. It's %s a year, risk-free with a 14-days money-back guarantee.",
+						{ args: premiumPlanPrice }
 					) }
```

Comment:
I am also concerned about translations here. Because we want to ship this ASAP, I want to make sure that we roll out the English strings quickly, but we should specify a good fallback for non-English locales. I don't see any relevant strings in Calypso at the moment, so maybe we should make the switch to the new text conditional for now.
```suggestion
import i18n, { useTranslate } from 'i18n-calypso';
// ...
			const currentLocale = i18n.getLocaleSlug();
// ...    
					{ currentLocale === 'en' || ( currentLocale && currentLocale.startsWith( 'en-') ) || i18n.hasTranslation( "Get access to our Premium themes, and a ton of other features, with a subscription to the Premium plan. It's {{strong}}%s{{/strong}} a year, risk-free with a 14-day money-back guarantee." )
						? translate(
							"Get access to our Premium themes, and a ton of other features, with a subscription to the Premium plan. It's {{strong}}%s{{/strong}} a year, risk-free with a 14-day money-back guarantee.",
							{
								args: { premiumPlanPrice  },
								components: { strong: <strong /> },
							}
						)
						: translate( 'You can purchase a subscription to use this theme or join the Premium plan to get it for free.' );
```
The gotcha here is that we then need copy for the pricing to make it clear that we're buying the Premium plan... 🤔

Reply:
> The gotcha here is that we then need copy for the pricing to make it clear that we're buying the Premium plan... :thinking:

cc: @SaxonF